apps/infrastructure/src/variables.tf

```diff
@@ -50,7 +50,7 @@ variable "cms_app_memory" {
 variable "use_custom_certificate" {
   type        = bool
   description = "Enable CDN https support with a custom certificate instead using the default one"
-  default     = false # set true when available dns and custom certificate
+  default     = true # set true when available dns and custom certificate
```

Comment:
**question (non-blocking)**: is the comment needed? or is the change needed?

I think the comment and the change are discordant

Reply:
ok, i've change the comment